app/javascript/components/widgets/weather/current-weather.jsx

```diff
@@ -32,6 +34,7 @@ const CurrentWeather = ({ weather, useLargeIcon }) => {
         <SmallSkyIcon icon={icon} />
       )}
       {description}
+      {Gust}
```

Comment:
Better to stick to regular JSX syntax here I think:

```jsx
Gust {weather.current.windGusts}
```

Reply:
Maybe we can inline this?
```jsx
{isGusty(weather)}
```

Reply:
or even better 
```jsx
{weather.current.windGust > 15 && ` , gusty wind at ${weather.current.windGust} mph`}
```

Reply:
Nit: prefer extracting/naming components over inlining code in JSX.